debian/changelog

```diff
@@ -1,3 +1,13 @@
+cloud-init (24.1.1-0ubuntu0) UNRELEASED; urgency=medium
```

Comment:
This is one of the few places that `new_upstream_snapshot.py` can guess wrong. Version number should be `24.1.1-0ubuntu1`.

Reply:
> This is one of the few places that new_upstream_snapshot.py can guess wrong

Is that something that can be fixed?

Reply:
Probably. I think in the past we always used `ubuntu/devel` even through feature freeze, so the usage is a little different.